examples/ChimneySwap/app/src/main/java/xyz/thepathfinder/www/chimneyswap/ParcelableChimney.java

```diff
@@ -0,0 +1,59 @@
+package xyz.thepathfinder.www.chimneyswap;
+
+import android.graphics.Bitmap;
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.google.android.gms.maps.model.LatLng;
+
+public class ParcelableChimney implements Parcelable {
```

Comment:
I was about to write a comment saying that you spelled "Parseable" wrong and then realized this is an android thing? lolandroid